lib/ts/recipe/oauth2provider/OAuth2Client.ts

```diff
@@ -49,6 +49,14 @@ export class OAuth2Client {
      */
     redirectUris: string[] | null;
 
+    /**
+     * Array of post logout redirect URIs
+     * StringSliceJSONFormat represents []string{} which is encoded to/from JSON for SQL storage.
+     *
+     * By default, this field is not present in the OAuth2Client. If provided, it must be a non-empty array of strings.
```

Comment:
Please add some info about what it "does" in each case (something like "if specified, the postLogoutRedirectUri must exactly match an item in the list, othewise the request is rejected", but I'm not sure that's correct and I'm sure it's not complete)

Reply:
Added more detailed comments.